.github/workflows/release.yml

```diff
@@ -12,6 +12,15 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
+        workflow_run:
+            workflows: ["Generate Manpages"]
+            types:
+              - completed
```

Comment:
I don't think workflow_run is supposed to be here? Maybe it should be a top-level thing, inside the `on` key.

Reply:
Ahh Thank you I wasn't 100% sure about the placement of this code!